app/assets/stylesheets/_maps.css.scss

```diff
@@ -0,0 +1,34 @@
+#map-canvas {
+  height: 300px;
+  width: 100%;
+}
+
+.controls {
+  border: 1px solid transparent;
+  border-radius: 2px 0 0 2px;
+  box-shadow: 0 2px 6px rgba(0, 0, 0, 0.3);
+  box-sizing: border-box;
+  height: 32px;
+  margin-top: 16px;
+  outline: none;
+}
+
+#pac-input {
+  background-color: $white;
+  font-size: 15px;
+  font-weight: 300;
+  margin-left: 12px;
+  padding: 0 11px 0 13px;
+  text-overflow: ellipsis;
+  width: 75%;
+
+  &:focus { border-color: #4d90fe; }
+}
+
+#type-selector {
+  background-color: #4d90fe;
+  color: $white;
+  padding: 5px 11px 0px 11px;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `5px 11px 0px` instead of `5px 11px 0px 11px`<br>`0px` should be written without units as `0`